From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: TB351FU <dev@tb351fu>
Date: Thu Jul 9 00:00:00 2026
Subject: [PATCH 4/5] Remove DebugLoggerUI from vendor configs

DebugLoggerUI is a diagnostic app that should not ship in
production ROMs.  Comment it out of vendor mk files so it is
not installed on the system image.

---
 vendor/lenovo/tb351fu/tb351fu-vendor.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/lenovo/tb351fu/tb351fu-vendor.mk b/vendor/lenovo/tb351fu/tb351fu-vendor.mk
index 822a71d..d3fafd0 100644
--- a/vendor/lenovo/tb351fu/tb351fu-vendor.mk
+++ b/vendor/lenovo/tb351fu/tb351fu-vendor.mk
@@ -25,7 +25,7 @@ PRODUCT_PACKAGES += \
     AccessibilityMenu \
     BatteryWarning \
-    DebugLoggerUI \
+    # DebugLoggerUI \
     LocationEM2 \
     MiraVision \
